### ISSUE TITLE
Support Angular and Angular Material

### ISSUE THREAD

**Issue by DragonDara** [abi/screenshot-to-code#164]
Dear Creator,

Do you have a long-term plan to implement support of Angular Material code generation from screenshots in the future?

**Comment by abi** [comment]
Yes, definitely would like to support as many integrations as possible. I'll be adding instructions on how people can add more integrations this week.

**Comment by AIxHunter** [comment]
Any updates on this ?
